docs/scenes/get-started/set-up-your-environment/SelectCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { ButtonBase, mergeClasses, useTheme } from '@expo/styleguide';
+import { useEffect, useState } from 'react';
+
+import { CALLOUT, HEADLINE } from '~/ui/components/Text';
+
+type Props = {
+  imgSrc: string;
+  darkImgSrc?: string;
+  alt: string;
+  title: string;
+  description?: string;
+  isSelected: boolean;
+  onClick: () => void;
+};
+
+export function SelectCard({
+  imgSrc,
+  darkImgSrc,
+  alt,
+  title,
+  description,
+  isSelected,
+  onClick,
+}: Props) {
+  const { themeName } = useTheme();
+  const [isDarkMode, setIsDarkMode] = useState(false);
+
+  useEffect(
+    function didMount() {
+      const darkMode =
+        themeName === 'dark' ||
+        (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches);
+      setIsDarkMode(darkMode);
+    },
+    [themeName]
+  );
+
+  return (
+    <ButtonBase onClick={onClick}>
+      <div className="w-[250px] shadow-xs overflow-hidden rounded-lg flex flex-col items-center border border-default hocus:scale-[102%] transition-all hocus:shadow-sm">
+        <div
+          className={mergeClasses(
+            'border-b border-default',
+            isSelected ? 'bg-gradient-to-b from-palette-blue3 to-palette-blue4' : 'bg-subtle'
+          )}>
+          <picture className="relative w-auto h-full flex items-end">
+            {isDarkMode && <source srcSet={darkImgSrc} type="image/png" />}
+            <img
+              src={imgSrc}
+              alt={alt}
+              className={mergeClasses(isSelected ? 'grayscale-0' : 'grayscale opacity-80')}
+            />
```

Comment:
Let's set the image sizes, or at least min-height to the container, so we can avoid lout shifts on render.